app/com/arpnetworking/metrics/portal/scheduling/JobExecutorActor.java

```diff
@@ -606,6 +652,23 @@ public Builder() {
         }
     }
 
+    /**
+     * Newtype wrapper around a {@link Reload} to mark its origin as the completion
+     * of a job, vs an anti-entropy tick or restart tick.
+     * @param <T>
+     */
+    private static final class ExecutionCompleted<T> {
+        private final Reload<T> _reload;
+
+        ExecutionCompleted(final Reload<T> reload) {
```

Comment:
I didn't put a builder here since it seemed verbose when I only intend for this to ever have a single field.